share/getters/options.go

```diff
@@ -0,0 +1,13 @@
+package getters
+
+import (
+	"github.com/celestiaorg/celestia-node/pruner"
+)
+
+type Option func(*ShrexGetter)
+
+func WithAvailabilityWindow(window pruner.AvailabilityWindow) func(*ShrexGetter) {
+	return func(s *ShrexGetter) {
+		s.availabilityWindow = window
+	}
+}
```

Comment:
I think it should be not an options, but part of constructor

Reply:
https://github.com/celestiaorg/celestia-node/pull/3188/commits/da20efc103bf02797140b067d0913396fc196089